lib/Qi/Model/Base.pm

```diff
@@ -0,0 +1,35 @@
+package Qi::Model::Base;
+use strict;
+use warnings;
+use Mojo::Base 'Mojolicious';
+use SQL::Abstract;
```

Comment:
With your code, there's no need for this.  You didn't call any of SQL::Abstract's functions directly here, you accessed the SQL::Abstract object from the main app, accessible via $self->sql.

However, you *do* want this line here because as you'll see on my L8 comment, you *will* access SQL::Abstract's methods directly (in this case, the "new" method).